### PR TITLE
AFKR-1: Added nutritional values and fee information sections

### PR DESCRIPTION
**Pull request by rubailly**
screencapture-192-168-33-10-bahmni-registration-index-html-2020-03-17-23_45_59

### CODE REVIEW

File: configuration/openmrs/apps/registration/extension.json [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+        "extensionParams": {
+            "conceptName": "Nutritional Values",
+            "translationKey": "NUTRITIONAL_VALUES_LOCALE_KEY",
+            "conceptNames": ["Height", "Weight", "BMI Data", "BMI Status Data"],
+            "required":true,
+            "showLatest": true
+        },
+        "order": 1,
+        "requiredPrivilege": "Edit Visits"
+    },
```
Comment (rbuisson) [inline configuration/openmrs/apps/registration/extension.json]: @rubailly  those are already provided in the Vital Signs section. Why twice?

Comment (rbuisson) [inline configuration/openmrs/apps/registration/extension.json]: @rubailly those are already provided in the Vital Signs section. Why twice?

File: configuration/openmrs/apps/registration/extension.json [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+        "id": "bahmni.registration.conceptSetGroup.vitalSigns",
+        "extensionPointId": "org.bahmni.registration.conceptSetGroup.observations",
+        "type": "config",
+        "extensionParams": {
+            "conceptName": "Haiti_Vital Signs Form",
+            "translationKey": "VITAL_SIGNS"
+        },
+        "order": 1,
+        "requiredPrivilege": "Edit Visits"
+    }
```
Comment (rbuisson) [inline configuration/openmrs/apps/registration/extension.json:70]: this should have `showLatest: true`

File: configuration/openmrs/i18n/registration/locale_fr.json [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+  "REGISTRATION_SMOKING_INJURIOUS_TEXT_KEY" : "Fumeurs , le tabac à chiquer peut causer le cancer , les maladies cardiaques et asthama .",
+  "REGISTRATION_INSTITUTE_NAME_TEXT_KEY": "Jan Swasthya Sahyog",
+  "REGISTRATION_INSTITUTE_REGISTERED_TEXT_KEY": "(Enregistré)",
+  "REGISTRATION_INSTITUTE_ADDRESS": "Ganiyari , District - Bilaspur",
+  "REGISTRATON_LATEST_KEY" : "Dernières",
+  "OBS_ABNORMAL_TRANSLATE_KEY":"Anormale",
+  "VITAL_SIGNS":"Signes vitaux",
+  "NUTRITIONAL_VALUES_LOCALE_KEY":"Valeurs nutritionnelles",
+  "TRIAGE_INFORMATION": "Informations de triage",
+  "FEE_INFORMATION_LOCALE_KEY":"Informations sur les frais"
```
Comment (rbuisson) [inline configuration/openmrs/i18n/registration/locale_fr.json]: Does not sound right in French I think. Let's make this "Frais".

**Review by rbuisson — CHANGES_REQUESTED** [review]
@rubailly , see review comments.

File: configuration/openmrs/i18n/registration/locale_fr.json (hunk 3 above)
Comment (rubailly) [reply configuration/openmrs/i18n/registration/locale_fr.json]: [Done](https://github.com/mekomsolutions/bahmni-config-easydoc/pull/5/commits/95a2e975d58baf345f2e4434e608b27194205070)

File: configuration/openmrs/apps/registration/extension.json (hunk 2 above)
Comment (rubailly) [reply configuration/openmrs/apps/registration/extension.json:70]: [Done](https://github.com/mekomsolutions/bahmni-config-easydoc/pull/5/commits/4c0ba5aea2f9cf0124c68657ea6ab904d73c5cfc)

File: configuration/openmrs/apps/registration/extension.json (hunk 1 above)
Comment (rubailly) [reply configuration/openmrs/apps/registration/extension.json]: [Done](https://github.com/mekomsolutions/bahmni-config-easydoc/pull/5/commits/4c0ba5aea2f9cf0124c68657ea6ab904d73c5cfc)

**Pull request merged**